packages/calcite-components/src/components/pagination/pagination.tsx

```diff
@@ -147,10 +148,29 @@ export class Pagination
     };
   }
 
-  @State() maxItems = maxItemBreakpoints.xsmall;
+  @State() maxItems = maxItemBreakpoints.xxsmall;
 
   @State() totalPages: number;
 
+  @State() lastStartItem: number;
+
+  @Watch("totalItems")
+  @Watch("pageSize")
+  @Watch("totalPages")
+  handleLastStartItemChange(): void {
+    const { totalItems, pageSize, totalPages } = this;
+
+    this.lastStartItem =
+      (totalItems % pageSize === 0 ? totalItems - pageSize : Math.floor(totalPages) * pageSize) + 1;
+  }
+
+  @State() isExtraExtraSmall: boolean;
+
+  @Watch("maxItems")
+  handleIsExtraExtraSmall(): void {
```

Comment:
Nitpick: can we rename `isExtraExtraSmall` and similar variables to `isXXSmall` to be slightly more concise and to match the breakpoint name?

This would also apply to the PR title.